_scripts/toolbox-create

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/bash -e
 
 echo "## Creating website container..."
-toolbox create -c cockpit-website
+toolbox create -c cockpit-website -r 32
```

Comment:
What's the reason for this (the commit message and/or a comment should point that out)? It'll get out of date quickly enough. Does f33 not work?

Reply:
F33 does not work currently. The idea is to bump it up on every stable release, so we have a known good platform instead of relying on the host system's version, which might be in flux (like it is currently).

It would be nice if we could just do `-r stable`, but I don't think that's a thing?